src/insights/competition.ts

```diff
@@ -3,13 +3,32 @@ import { PageMeta, TimeMeta } from "./common";
 export interface CompetitionItem {
   id: string;
   competitionId: string;
+  subCompetitionId: string;
   market: string;
   start: string; // string representation of timestamp
   end: string; // string representation of timestamp
   enablePNL: boolean;
   minBalance: number;
   enableVolume: boolean;
   makerIncentive: boolean;
+  config: CompetitionItemConfig;
+}
+
+export interface CompetitionItemConfig {
+  type: string;
+  lending: string[];
+  markets: string[];
+  promoMarkets: string[];
+  preTradeMarket: string[];
+  minBalance: number;
+  volPerPoint: number;
+  allowDecimals: boolean;
+  derMultiplier: number;
+  lendingQuantity: number;
```

Comment:
`derMultiplier` and `lendingQuantity` are both number strings
<img width="203" alt="Screenshot 2024-07-10 at 5 33 10 PM" src="https://github.com/Switcheo/carbon-js-sdk/assets/65583522/3c22ce12-cf12-42ae-b123-fd7528b290bd">